LEGO1/mxpalette.cpp

```diff
@@ -1,4 +1,349 @@
 #include "mxpalette.h"
+#include "mxomni.h"
+
+PALETTEENTRY g_defaultPaletteEntries[256] =
```

Comment:
I assume this is a global/static table? Should list address here.

Reply:
What format should the comment be in?

Reply:
So far we've just been listing the address as a comment for documentation (`// 0xDEADBEEF`), however the format it needs is probably `// GLOBAL: <MODULE> <ADDRESS> <SIZE>` so the actual values can be compared with `reccmp`.

Reply:
I looked through the recmp.py code and i dont know if what I just pushed is sufficient

Reply:
`reccmp` doesn't have the functionality yet, this is just what I think will be necessary to achieve the verification.

Sizes should be in hex too, but I'll fix that so this can be merged.